### DISCUSSION TITLE
Dask-unhashable type error

### DISCUSSION THREAD

**Discussion by OMERTBITU** [blue-yonder/tsfresh#1052] [Q&A]
Hello I am trying to use rolled type feature extractor with dask dataframe for faster implementation with following code:

tsfresh_df=pd.DataFrame(tsfresh_son_np,columns=["id","time","csr"]) 
from tsfresh.utilities.dataframe_functions import roll_time_series 
df_rolled = roll_time_series(tsfresh_df, column_id="id", column_sort="time",max_timeshift=20) 
from tsfresh import extract_features tsfresh_rolled_dask=dd.from_pandas(df_rolled, npartitions=3) 
df_features =extract_features(tsfresh_rolled_dask, column_id="id", column_sort="time",pivot=False,show_warnings=False)
result = df_features.compute()

But it gives error as: TypeError: unhashable type: 'list'

What is the problem?

**Comment by nils-braun** [comment]
Hi @OMERTBITU !

Can you share the full stacktrace? It will help find the root cause of the issue.
Thanks!

(also, could you write your example code with code formatting? It helps with readability a lot :) )
